__tests__/sagas/nanoContracts/fixtures.js

```diff
@@ -0,0 +1,128 @@
+import { jest } from '@jest/globals';
+
+export const fixtures = {
+  address: 'HTeZeYTCv7cZ8u7pBGHkWsPwhZAuoq5j3V',
+  ncId: '3cb032600bdf7db784800e4ea911b10676fa2f67591f82bb62628c234e771595',
+  ncApi: {
+    getNanoContractState: {
+      successResponse: {
+        success: true,
+        nc_id: '3cb032600bdf7db784800e4ea911b10676fa2f67591f82bb62628c234e771595',
+        blueprint_name: 'Bet',
+        fields: {
+          token_uid: { value: '00' },
+          total: { value: 300 },
+          final_result: { value: '1x0' },
+          oracle_script: { value: '76a91441c431ff7ad5d6ce5565991e3dcd5d9106cfd1e288ac' },
+          'withdrawals.a\'Wi8zvxdXHjaUVAoCJf52t3WovTZYcU9aX6\'': { value: 300 },
+          'address_details.a\'Wi8zvxdXHjaUVAoCJf52t3WovTZYcU9aX6\'': { value: { '1x0': 100 } },
+        },
+      },
+    },
+    getNanoContractHistory: {
+      failureResponse: {
+        success: false,
+      },
+      errorResponse: {
+        error: new Error('API call error'),
+      },
+      successResponse: {
+        success: true,
+        history: [
+          {
+            hash: '000000203e87e8575f121de16d0eb347bd1473eedd9f46cc76c1bc8d4e5a5fce',
+            nonce: 2104638,
+            timestamp: 1708356261,
+            version: 4,
+            weight: 21.89480540500889,
+            signal_bits: 0,
+            parents: [
+              '0000008fbebdf8d78be50c88aceebf3c6b9e92f4affd7dfc96d48a7a49f23e69',
+              '00000121c46366b19de5efa8e6c23f62895322486395a0e31e987f9073025989'
+            ],
+            inputs: [
+              {
+                tx_id: '0000008fbebdf8d78be50c88aceebf3c6b9e92f4affd7dfc96d48a7a49f23e69',
+                index: 0,
+                data: 'RjBEAiBtWa0q8uzMvBfkh83Y+t4Tv5OeyJSD8NazaGp19Hc2UwIgbV2m5unBEHlTAcLJsZLsCBlnfpua8LrUkVORiW/t4OQhAolqAR4yFaeCBeu/kOG1SwWnRj2X62zT9mU+Deutnbqq'
+              }
```

Comment:
Is there ana possibility to return an [`IHistoryInput`](https://github.com/HathorNetwork/hathor-wallet-lib/blob/54b250c71d03d18b80b228b61b0604bba3fabb53/src/types.ts#L78)?

Reply:
What do you mean? Is this comment in the right lines?